ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/node-type-l3-port-channels.yml

```
@@ -0,0 +1,164 @@
+              name: Port-Channel5
+              mode: passive
+              member_interfaces:
+                - name: Ethernet1/4
+                  peer: peerDevice2
+                  peer_interface: Ethernet1/12
+                  speed: "forced 10000full"
+                - # if peer not set, use parent L3 Port-Channel peer
+                  name: Ethernet1/5
+                  speed: "forced 10000full"
+              ip_address: 192.168.1.105/31
+              peer: peer2
+              peer_port_channel: Port-Channel15
+              peer_ip: 192.168.1.15
+              qos_profile: TEST-QOS-PROFILE1
+              wan_carrier: StreamFast
+              wan_circuit_id: 102
+              flow_tracking:
+                enabled: false
+              structured_config:
+                service_policy:
+                  qos:
+                    input: TEST_POLICY
+              raw_eos_cli: |
+                ! TEST RAW_EOS_CLI
+            - # sub-interface for Port-Channel with user-specfied encapsulation_dot1q_vlan
+              name: Port-Channel5.100
+              encapsulation_dot1q_vlan: 108
+              ip_address: 192.168.100.115/31
+              peer: peer2
+              peer_port_channel: Port-Channel15
+              peer_ip: 192.168.1.15
+              qos_profile: TEST-QOS-PROFILE2
+              wan_carrier: ExtremeCable
+              wan_circuit_id: 105
+              flow_tracking:
+                enabled: true
+            - # sub-interface for Port-Channel without wan_carrier set
+              name: Port-Channel5.105
+              ip_address: 192.168.100.116/31
+              peer: peer2
+              peer_port_channel: Port-Channel16
+              peer_ip: 192.168.1.16
+              qos_profile: TEST-QOS-PROFILE2
+              flow_tracking:
+                enabled: true
+            - # Port-Channel with 1 member port
+              name: Port-Channel8
+              mode: 'on'
+              member_interfaces:
+                - name: Ethernet1/10
+                  peer: peerDevice3
+                  peer_interface: Ethernet1/10
+                  structured_config:
+                    # specify interface speed via structured_config
+                    speed: "forced 1000full"
+              peer: peerDevice3
+              peer_port_channel: Port-Channel18
+              peer_ip: 192.168.1.18
+              qos_profile: TEST-QOS-PROFILE3
+              wan_carrier: BlizzardFast
+              ip_address: dhcp
+              dhcp_ip: 10.15.16.17
+              dhcp_accept_default_route: true
+              flow_tracking:
+                enabled: true
+              raw_eos_cli: |
+                ! TEST RAW_EOS_CLI 123
+            - # Port-Channel interface is shutdown (`enabled: false`), member ports to inherit this.
+              # Port-Channel mode is not set.
+              name: Port-Channel19
+              enabled: false
+              ip_address: 192.168.1.19
```

hmm it should have a mask or it will fail.
```suggestion
              ip_address: 192.168.1.19/31
```